frontend/test/metabase/scenarios/models/models-query-editor.cy.spec.js

```diff
@@ -89,13 +92,15 @@ describe("scenarios > models query editor", () => {
       .findByText("Summarize")
       .click();
     selectFromDropdown("Count of rows");
+
+    cy.wait("@dataset");
```

Comment:
I'm constantly seeing this failure:
![image](https://user-images.githubusercontent.com/31325167/152158256-8fd50326-f6ad-492a-b4f2-47c8be06759b.png)